input/standardized-verification/notebook.ipynb

```diff
@@ -0,0 +1,1518 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Import Necessary Module and Set Path "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sqlite3 as lite\n",
+    "import os\n",
+    "from IPython.display import Image\n",
+    "import sys\n",
+    "import numpy as np\n",
+    "import collections\n",
+    "sys.path.insert(0, '../../scripts/')\n",
+    "jupyter_directory = os.getcwd()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import analysis as an"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Must use Cycamore in jbae11 cycamore branch \"standardized_verif\"\n",
+    "\n",
+    "`git clone https://github.com/jbae11/cycamore -b standardized_verif`\n",
+    "\n",
+    "### Changes from cyclus/cycamore master:\n",
+    "1. Reactor depletes all core rather than half core\n",
+    "2. Cycamore::Separations and Storage::Storage outputs extra tables"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## ISSUES\n"
```

Comment:
The cell following the somewhat scary "ISSUES" heading is completely blank. The reader is now concerned about mystery issues.